org.freedesktop.Sdk.Extension.php73.json

```diff
@@ -22,9 +22,56 @@
 		"append-pkg-config-path": "/usr/lib/sdk/php73/lib/pkgconfig"
 	},
 	"modules": [
+		{
+			"name": "libldap",
+			"buildsystem": "autotools",
+			"config-opts": [
+				"--disable-slapd",
+				"--disable-static"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.48.tgz",
+					"sha256": "d9523ffcab5cd14b709fcf3cb4d04e8bc76bb8970113255f372bc74954c6074d"
+				},
+				{
+					"type": "patch",
+					"path": "openldap-aarch64.patch"
```

Comment:
If this is only useful for aarch64, you can use         `"only-arches": "aarch64"` here